src/crypto/ocb_internal.cc

```diff
@@ -679,7 +709,7 @@ static void process_ad(ae_ctx *ctx, const void *ad, int ad_len, int final)
 				ad_offset = xor_block(oa[6], getL(ctx, tz));
 				ta[7] = xor_block(ad_offset, adp[7]);
 			#endif
-			AES_ecb_encrypt_blks(ta,BPI,&ctx->encrypt_key);
+			ocb_aes::ecb_encrypt_blks(ta,BPI,&ctx->encrypt_key);
```

Comment:
Put spaces after all commas that are missing spaces on lines that were modified? It seems in general this file uses space-after-comma except ... for when it doesn't.

Reply:
Done.